balancer-js/examples/flashSwap.ts

```diff
@@ -0,0 +1,88 @@
+import Web3 from 'web3';
```

Comment:
We're using ethers in the other examples. What do you think about keeping it consistent and using it here too?

Reply:
Yeah, I don't think it'll be a problem, although there's probably a benefit to having both examples -- web3 and ethers